website/docs/r/iothub.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iothub"
+sidebar_current: "docs-azurerm-resource-iothub"
+description: |-
+  Creates a new IotHub resource 
+---
+
+# azurerm\_iothub
+
+Creates a new IotHub
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "resourceGroup1"
+  location = "West US"
+}
+
+
+resource "azurerm_iothub" "test" {
+	name                             = "test"
+	resource_group_name              = "${azurerm_resource_group.test.name}"
+	location                         = "westus"
```

Comment:
_minor_ could we make this reference the location field on the resource group, e.g.

```
location = "${azurerm_resource_group.test.location}"
```